### DISCUSSION TITLE
What browsers are used in development?

### DISCUSSION THREAD

**Discussion by markfirmware** [alpine-collective/alpine-magic-helpers#64] [Q&A] [answered]
I use firefox but I might switch.

**Comment by SimoTod** [comment, marked as answer]
I personally use chrome but the library should work with any modern browsers so it's actually better if other people use different one so it's more likely that they will find and report bugs

**Reply by markfirmware** [reply]
Ok, I will continue with firefox

**Comment by KevinBatdorf** [comment]
I use Chrome too. I've sometimes switched to Firefox but I quickly switch back. I guess it's just familiar at this point.
